### ISSUE TITLE
deprecation warning when using talbe steps

### ISSUE THREAD

**Issue by bitboxer** [makandra/spreewald#48]
I have a few deprecation warnings when using the table steps helper:

```
DEPRECATION: `failure_message_for_should` is deprecated. Use `failure_message` instead. Called from /Users/bodo/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/spreewald-1.2.6/lib/spreewald/table_steps.rb:76:in `block in <module:TableStepsHelper>'.
DEPRECATION: `failure_message_for_should_not` is deprecated. Use `failure_message_when_negated` instead. Called from /Users/bodo/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/spreewald-1.2.6/lib/spreewald/table_steps.rb:86:in `block in <module:TableStepsHelper>'.
DEPRECATION: Using `should` from rspec-expectations' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` with `config.expect_with(:rspec) { |c| c.syntax = :should }` instead. Called from /Users/bodo/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/spreewald-1.2.6/lib/spreewald/table_steps.rb:122:in `block (2 levels) in <top (required)>'.
```

**Comment by judithroth** [comment]
The first two deprecation warnings are already fixed in 1.12.2.

**Comment by judithroth** [comment]
The other deprecation warnings should be fixed with ce3aa975438647c0db3b05bb426c3a6d6d29bda2

**Comment by bitboxer** [comment]
Thanks for fixing this.
